### PR TITLE
Delete study functionality

### PR DESCRIPTION
**Pull request by squirrelo**
Depends on #1589 

This adds the deletion of studies to the new layout.

### CODE REVIEW

**Comment by antgonza** [comment]
@squirrelo could you pull from the branch and fix the conflicts? Thanks.

**Comment by squirrelo** [comment]
done.

**Comment by antgonza** [comment]
:+1:

**Comment by squirrelo** [comment]
ping @josenavas

File: qiita_pet/handlers/api_proxy/tests/test_studies.py [hunk 1]
```diff
@@ -101,5 +102,49 @@ def test_study_prep_get_req_no_access(self):
                'message': 'User does not have access to study'}
         self.assertEqual(obs, exp)
 
+    def test_study_delete_req(self):
```
Comment (josenavas) [inline qiita_pet/handlers/api_proxy/tests/test_studies.py]: This is modifying the DB - shouldn't the class be decorated with the safeguard that we put in place?

File: qiita_pet/templates/study_base.html [hunk 2]
```diff
@@ -126,11 +152,11 @@
   <h3><a href="#" onclick="fill_base({{study_info['study_id']}})">Study Information</a></h3>
     <a href="#" onclick="fill_sample({{study_info['study_id']}}); return false;">Sample Template</a><br />
     <a href="#" onclick="fill_sample_summary({{study_info['study_id']}}); return false;">Sample Summary</a><br />
-    {% if editable %}
+    {# if editable #}
```
Comment (josenavas) [inline qiita_pet/templates/study_base.html]: why this change from % to #?

File: qiita_pet/templates/study_base.html [hunk 3]
```diff
@@ -112,11 +112,37 @@
...
+    if ($("#study-alias").val() == "{{study_info['study_alias']}}") {
+      $('#delete-study-button').prop('disabled', false);
+    } else {
+      $('#delete-study-button').prop('disabled', true);
+    }
+  }
+
+function delete_study() {
+  if ($("#study-alias").val() != "{{study_info['study_alias']}}") {
+    alert("The added name doesn't match the study alias");
```
Comment (josenavas) [inline qiita_pet/templates/study_base.html]: This reads a bit weird. Change "The added name" by "The provided alias", or "The given alias", but it is not added.

**Comment by josenavas** [comment]
Few comments

File: qiita_pet/templates/study_base.html (hunk 2 above)
Comment (squirrelo) [inline qiita_pet/templates/study_base.html]: test that got through.

File: qiita_pet/templates/study_base.html (hunk 3 above)
Comment (squirrelo) [inline qiita_pet/templates/study_base.html]: fixed

File: qiita_pet/handlers/api_proxy/tests/test_studies.py (hunk 1 above)
Comment (squirrelo) [inline qiita_pet/handlers/api_proxy/tests/test_studies.py]: Done
